dashboard/test/helpers.js

```diff
@@ -0,0 +1,86 @@
+const chain = require('chain-sdk')
+const client = new chain.Client()
+
+global.sleep = (ms) => {
```

Comment:
This isn't about promise fulfillment so much as giving `cored` time to start up; the promises will resolve or reject rather quickly as is. Rather, I want calls _after_ `client.config.configure({})` to wait a second before being attempted.

Something like https://github.com/magicdawn/promise.sleep matches this case better. How would you feel about a promise that resolves with setTimeout instead?

```js
const sleep = (ms) => new Promise(resolve => {
  setTimeout(() => resolve(), ms)
}) 
```